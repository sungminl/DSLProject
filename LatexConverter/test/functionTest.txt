&a = 1
&b = 99
FUNCTION print_Text (&a, &b) {
&c = [1, 2, 3]
TEXT(Bold, Underline)
{
"print this"
}
MATRIX
COL: 3, ROW: 3(&a,&b,&c)
{
&c|
&a, 1, &b|
&c * 3|
}
}

CALL(print_Text, &a, &b)