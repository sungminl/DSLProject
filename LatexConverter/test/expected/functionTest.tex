\documentclass[]{article}
\usepackage{amsmath}
\usepackage{tikz}
\usetikzlibrary{graphs,automata}
\title{Document}
\begin{document}
\textbf{\underline{print this}}

\[\begin{bmatrix}
1.0 & 2.0 & 3.0\\
1.0 & 1.0 & 99.0\\
3.0 & 6.0 & 9.0\\
\end{bmatrix}\]
\end{document}
